src/standard/gasometer/mod.rs

```diff
@@ -202,6 +211,11 @@ impl GasometerState {
 			MergeStrategy::Discard => {}
 		}
 	}
+
+	/// Apply the floor gas cost for calldata as defined in EIP-7623
+	pub fn apply_transaction_floor_cost(&mut self) {
+		self.used_gas = max(self.used_gas, self.floor_gas);
+	}
```

Comment:
Where is this being used?

Reply:
Added config and invocation here: https://github.com/rust-ethereum/evm/pull/391/commits/bb70b3839a83256d26c27a84cab61f96ecd8d189